tests/Gating.UnitTests/GateContextUnitTests.cs

```diff
@@ -21,6 +21,11 @@ namespace Microsoft.Omex.Gating.UnitTests
 	/// </summary>
 	public sealed class GateContextUnitTests : UnitTestBase
 	{
+#pragma warning disable SM02239 // hard coded e-mail for unit tests
+		private static readonly string s_userIdentifierInGroup = "test1@microsoft.com";
+		private static readonly string s_userIdentifierNotInGroup = "test1@apple.com";
```

Comment:
I'd suggest using @contoso.com, given that this is a Microsoft address.

Reply:
Updated, thank you